### ISSUE TITLE
You have to click twice on an action to delete (using keyboard)

### ISSUE THREAD

**Issue by kisubedi** [Azure/LogicAppsUX#3527]
### Describe the Bug with repro steps

Simple repro

### What type of Logic App Is this happening in?

Consumption (Portal)

### Are you using new designer or old designer

New Designer

### Did you refer to the TSG before filing this issue? https://aka.ms/lauxtsg

Yes

### Browser

Chrome

AB#25598018

**Comment by hartra344** [comment]
I get that this is a simple repro but @kisubedi for issues you open, please fill out the form completely with repro steps and video of the issue. It does make things easier for us.

**Comment by Elaina-Lee** [comment]
Unable to reproduce this on standalone/portal on LogicApps

**Comment by hartra344** [comment]
@kisubedi I also don't really understand this one. Please provide more information. I'm marking this as needs more info so it'll auto close in 14-21 days otherwise.
